corehq/motech/repeaters/tests/test_models_slow.py

```diff
@@ -47,11 +47,13 @@ def setUpClass(cls):
             include_app_id_param=False,
         )
         cls.repeater.save(sync_to_sql=False)
-        cls.sql_repeater = SQLRepeater.objects.create(
+        cls.sql_repeater = SQLFormRepeater(
             domain=DOMAIN,
             repeater_id=cls.repeater.get_id,
             connection_settings=conn,
+            format='form_xml',
```

Comment:
I actually made the `FORMAT_OPTIONS` local to every Repeater class, which would allow us this.

Reply:
I was asking about passing `format=...` to `SQLFormRepeater` since `format` is an `OptionValue` property, so the Django ORM does not know about it, and the default `__init__` will raise `TypeError: SQLFormRepeater() got an unexpected keyword argument 'format'`

Reply:
Ohh! This did not throw any error and tests ran fine. Can it be because, `SQLFromRepeater` is a proxy model and when it is actually translated to a SQLRepeater instance the other attributes were moved to options attribute?